sig-network/role-sig-chair.md

```diff
@@ -0,0 +1,70 @@
+# Role: SIG Chair
+
+## Background
+
+Please see the
+[standard description of a SIG chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair),
+which does a good job describing the expectations of the more “administrative”
+aspect of being a SIG chair.
+
+## What's expected
+
+Rather than repeat the doc linked above, this doc will just add some color.
+
+The minimum effective engagement for a SIG chair is to “run the show” (really
+“make sure the show is running” - delegation is good!).  That generally means
+building an agenda for and running regular meetings, curating our
+community-facing metadata, and acting as liaison to other groups (other SIGs
+and steering, mostly).
+
+This includes new-issue triage - either doing the triage or leading it (e.g. in
```

Comment:
issues and PRs? Or just issues?

I think this also needs to include tracking the trends of open/closed issues.

Reply:
trends are important to watch to make sure the list is being worked on. Bugs can be triaged, but nobody is working on them